packages/x-date-pickers/src/DateField/tests/editing.DateField.test.tsx

```diff
@@ -744,6 +744,65 @@ describe('<DateField /> - Editing', () => {
     });
   });
 
+  describeAdapters('Disabled field', DateField, ({ renderWithProps }) => {
+    it('should not key editing on disabled field', () => {
+      // Test with v7 input
+      const onChangeV7 = spy();
+      const v7Response = renderWithProps({
+        enableAccessibleFieldDOMStructure: true,
+        onChange: onChangeV7,
+        disabled: true,
+      });
+
+      const keys = [
+        'ArrowUp',
+        'ArrowDown',
+        'PageUp',
+        'PageDown',
+        'Home',
+        'End',
+        'Delete',
+        'ArrowLeft',
+        'ArrowRight',
+        'Meta',
+        'Control',
+      ];
+
+      v7Response.selectSection('month');
+
+      keys.forEach((key) => {
+        v7Response.pressKey(0, key);
+        expectFieldValueV7(v7Response.getSectionsContainer(), 'MM/DD/YYYY');
+        expect(onChangeV7.callCount).to.equal(0);
+      });
+
+      // digit key press
+      // v7Response.pressKey(0, '2');
```

Comment:
This event trigger still seems to be able to change the input value 🤯 
I'm investigating it, but accept suggestions 👼